libs/design/src/molecules/callout/callout/callout.component.scss

```diff
@@ -13,14 +13,13 @@
   &__title {
     @include embolden();
     font-size: 1.5rem;
-    text-align: center;
+    text-align: left;
```

Comment:
I'm not certain I understand why this was changed, my prior comments were in regards to generically setting `text-align:center` all transcluded content inside a hero/callout via `> *`.

Reply:
I decided the default hero/callout aligns should just be left aligned even on mobile.